### DISCUSSION TITLE
Idea: map idea. Gladiator arena.

### DISCUSSION THREAD

**Discussion by ShiroWork-s** [redeclipse/base#1265] [Ideas and Feature Requests]
(I couldnt decide to put this in ideas or map.. so I believe ideas is the better option because i'm not showcasing any map. I'm not familiar enough with the game map creation to create this, but how about it? 
In my opinion, and just my opinion.. Any fps game needs an onslaught arena map. A close cornered yet open arena that has good sights and even better speed. Imagine this: 
An arena with high walls and even higher stakes, with crowds watching (just art or posters, definitely not entities) the arena's participants battle it out in an onslaught gamemode.
The design could be like any classic arena, like in the roman's style. But the color palette would be different, like the new tech colors the game is styled in.
One side could have open areas full of sand or something,
another side could have small cover that allows shotgun combat.
How about it?
:/ more I inspect it, the more complicated it want it to become... It should stay simple as a fun map labeled for onslaught that everyone can enjoy.

**Comment by qreeves** [comment]
We had quite a few gladiator maps in the previous iteration of Red Eclipse, but many haven't been ported yet as we're still low on regular maps.

**Reply by ShiroWork-s** [reply]
Cool.
How do you make your maps from scratch?
Im thinking of making one from scratch, but i dont know where to start.

**Reply by SniperGoth** [reply]
You can begin a editing match straight up, by typing /edit whatevernameofthemap, and that's pretty much it.
F2  to toggle between editing and playing, F3 to open the editing menu(which has a lot of things to play around with), scroll wheel to pull or push cubes, and so on.
Someone can go online in a server with you to teach everything you need, if you want.
